common/tools/configtxgen/encoder/encoder.go

```diff
@@ -109,6 +109,12 @@ func addImplicitMetaPolicyDefaults(cg *cb.ConfigGroup) {
 	addPolicy(cg, policies.ImplicitMetaAnyPolicy(channelconfig.WritersPolicyKey), channelconfig.AdminsPolicyKey)
 }
 
+// addOrdererImplicitMetaPolicyDefaults adds the orderer's Readers/Writers/Admins/BlockValidation policies, with Any/Any/Majority/Any rules respectively.
+func addOrdererImplicitMetaPolicyDefaults(cg *cb.ConfigGroup) {
+	addPolicy(cg, policies.ImplicitMetaAnyPolicy(BlockValidationPolicyKey), channelconfig.AdminsPolicyKey)
```

Comment:
We can't use addPolicy here because it uses the same key name as the sub policy. We can just copy the removed Policies map addition below here